kevm-pyk/src/kevm_pyk/kproj/evm-semantics/serialization.md

```diff
@@ -21,7 +21,7 @@ Address/Hash Helpers
 -   `keccak` serves as a wrapper around the `Keccak256` in `KRYPTO`.
 
 ```k
-    syntax Int ::= keccak ( Bytes ) [function, total, smtlib(smt_keccak)]
+    syntax Int ::= keccak ( Bytes ) [function, total, injective, smtlib(smt_keccak)]
  // -------------------------------------------------------------------------
```

Comment:
```suggestion
    syntax Int ::= keccak ( Bytes ) [function, total, injective, smtlib(smt_keccak)]
 // ------------------------------------------------------------------------------------
```

Reply:
How strange, I thought this was taken care of...